Error: ENOENT: no such file or directory, open '/tmp/test-plugins1/config'

Failing with `npm test`:

```
Error: ENOENT: no such file or directory, open '/tmp/test-plugins1/config'
    at Object.openSync (fs.js:436:3)
    at Object.readFileSync (fs.js:341:35)
    at writePluginConfig (/home/christianbundy/src/ssbc/scuttlebot/plugins/plugins.js:64:42)
    at /home/christianbundy/src/ssbc/scuttlebot/plugins/plugins.js:132:19
    at /home/christianbundy/src/ssbc/scuttlebot/node_modules/mv/index.js:34:26
    at FSReqWrap.oncomplete (fs.js:141:20)
npm ERR! Test failed.  See above for more details.
``

--- a/plugins/plugins.js
+++ b/plugins/plugins.js
@@ -61,14 +61,28 @@ module.exports = {
     function writePluginConfig (pluginName, value) {
       var cfgPath = path.join(config.path, 'config')
       // load ~/.ssb/config
-      var existingConfig = JSON.parse(fs.readFileSync(cfgPath, 'utf-8'))
+      let existingConfig
+      fs.readFile(cfgPath, 'utf-8', (err, data) => {
+        if (err) {
+          if (err.code === 'ENOENT') {
+            // only catch "file not found"
+            existingConfig = {}
+          } else {
+            throw err
+          }
+        } else {
+          existingConfig = JSON.parse(data)
+        }
 
-      // update the plugins config
-      existingConfig.plugins = existingConfig.plugins || {}
-      existingConfig.plugins[pluginName] = value
 
-      // write to disc
-      fs.writeFileSync(cfgPath, JSON.stringify(existingConfig, null, 2), 'utf-8')
+        // update the plugins config
+        existingConfig.plugins = existingConfig.plugins || {}
+        existingConfig.plugins[pluginName] = value
+
+        // write to disc
+        fs.writeFileSync(cfgPath, JSON.stringify(existingConfig, null, 2), 'utf-8')
+      })
+
     }
 
     return {

